verticapy/tests/vDataFrame/test_vDF_plot.py

```diff
@@ -228,6 +228,7 @@ def test_vDF_bar(self, titanic_vd, amazon_vd):
         )
         plt.close("all")
 
+    @pytest.mark.skipif(sys.version_info >= (3, 7), reason="this test is incompatible with newer versions of matplotlib")
```

Comment:
The matplotlib installed in the test environment for Python 3.7 and 3.8 is 3.5.0 which was released on November 15, 2021. The way several tests in this file are written is incompatible with this new version of the package. The environment installs matplotlib 3.3.4 for Python 3.6, so these tests are still passing for that version.
We should improve these tests not to rely on internal implementation of the matplotlib package.